bing.com.BingWebMasterDPOO.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOO",
```

Comment:
I don't like the mixed case serviceId name.  Same with the file name.  Can we use bing.com.dpoo.json?  This is what the other microsoft template did for o365.

Reply:
@arnoldblinn with the current language in the spec this field is case-insensitive anyway, so shall we bother anymore?

Reply:
Indeed, the values pass in the query string for the provider/service names are case insensitive. Largely because people started doing it.  But for template names, they are case sensitive.  All other template names are lower case, making the mapping of the queyr string values to a file name easy.  I do think we should make this lower case.

Reply:
Done.